### DISCUSSION TITLE
Discord Bot

### DISCUSSION THREAD

**Discussion by rafaelmaiach** [dailydotdev/daily#1009] [Feature Requests]
Is it possible to use this API just to consume and post the news on a Discord Server?

**Comment by idoshamun** [comment]
Hey @rafaelmaiach,

Unfortunately, we still do not offer public access to our API.
But we do consider creating a discord/slack bot ourselves.
Stay tuned.

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
